app/helpers/mail_helper.rb

```diff
@@ -0,0 +1,47 @@
+module MailHelper
+
+  APP_GUIDANCE_PAGE = 'https://intranet.justice.gov.uk/peoplefinder'.freeze
+
+  def browser_warning
+    content_tag(:p, class: 'browser-warning') do
+      mailer_t(:browser_warning)
+    end
+  end
+
+  def easy_copy_link_to url:
```

Comment:
Could you put the arguments inside parentheses, please?  This style isn't consistent with the rest of the commit or how we normally do it...